patroni/api.py

```diff
@@ -542,7 +543,9 @@ def __initialize(self, config):
         # Sometime it's also needed to pass reference to a 'keyfile'.
         if self.__ssl_options.get('certfile'):
             import ssl
-            self.socket = ssl.wrap_socket(self.socket, server_side=True, **self.__ssl_options)
+            ctx = ssl.create_default_context(ssl.Purpose.CLIENT_AUTH)
+            ctx.load_cert_chain(certfile=self.__ssl_options['certfile'], keyfile=self.__ssl_options.get('keyfile'))
```

Comment:
If you remove `ciphers` from line 515 you can change this line to:
```python
ctx.load_cert_chain(**self.__ssl_options)
```

If you still want to explicitly specify `ciphers` just get the value from the `config`.

Reply:
Updated. Thanks!